### DISCUSSION TITLE
Generating a Java client library

### DISCUSSION THREAD

**Discussion by k4nd1l** [amzn/selling-partner-api-models#3736] [General] [answered]
After running the `mvn package` command to create the swagger jar , it gives me a compilation error `package javax.annotation does not exist`. How could I solve this?

![Screenshot 2023-11-14 160718](https://github.com/amzn/selling-partner-api-docs/assets/47002194/555fab21-cf7e-4083-ae8a-c74728a94400)

**Comment by ayoubAnbara** [comment, marked as answer]
Add the dependency [javax.annotation](https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api) to the project.
For more details, look this article https://jesseevers.com/spapi-first-application/
